datacommons/core.py

```diff
@@ -95,33 +93,36 @@ def get_property_values(dcids,
   payload = utils._format_response(res)
 
   # Create the result format for when dcids is provided as a list.
-  result = defaultdict(list)
+  results = defaultdict(list)
   for dcid in dcids:
+    # Make sure each dcid is mapped to an empty list.
+    results[dcid]
+
+    # Add elements to this list as necessary.
     if dcid in payload and prop in payload[dcid]:
       for node in payload[dcid][prop]:
         if 'dcid' in node:
-          result[dcid].append(node['dcid'])
+          results[dcid].append(node['dcid'])
         elif 'value' in node:
-          result[dcid].append(node['value'])
-    else:
-      result[dcid] = []
+          results[dcid].append(node['value'])
 
-  # Format the result as a Series if a Pandas Series is provided.
+  # Format the results as a Series if a Pandas Series is provided.
   if isinstance(dcids, pd.Series):
-    return pd.Series([result[dcid] for dcid in dcids])
-  return dict(result)
+    return pd.Series([results[dcid] for dcid in dcids])
+  return dict(results)
 
 
 def get_triples(dcids, limit=utils._MAX_LIMIT):
-  """ Returns a list of triples where the dcid is either a subject or object.
+  """ Returns all triples associated with the given dcids.
 
-  The return value is a list of tuples (s, p, o) where s denotes the subject
-  entity, p the property, and o the object.
+  The return value is a dictionary mapping given dcids to list of triples. The
+  triples are repsented as 3-tuples (s, p, o) where "s" denotes the subject, "p"
+  the property, and "o" the object.
 
   Args:
-    dcid: A list of dcids to get triples for.
+    dcids: A list of dcids to get triples for.
     limit: The maximum number of triples to get for each combination of property
```

Comment:
can say property label to be more clear

Reply:
Will clarify with a section on what a triple is.